src/midiEditorCommands.cpp

```diff
@@ -994,6 +994,55 @@ void cmdMidiFilterWindow(Command *command) {
 	}
 }
 
+void maybePreviewCurrentNoteInEventList(HWND hwnd) {
+	auto focused = ListView_GetNextItem(hwnd, -1, LVNI_FOCUSED);
+	char text[50];
```

Comment:
If you're worried about this macro failing and not touching the string (which it seems you might be given the nulling later), you might want to ensure the first char of text is null here, since you're not explicitly zeroing it currently:
`char text[50] = "\0";`